### DISCUSSION TITLE
How query "like" query ?

### DISCUSSION THREAD

**Discussion by wildwind123** [siglens/siglens#745] [Q&A]
I write log with vector, and log written in message column.

![Screenshot from 2024-04-13 13-47-38](https://github.com/siglens/siglens/assets/34130873/bb9770b5-1ebd-44d1-820f-ade5d750c06a)

And want to query like this. 
```
select * from table where message like '%Watching movies%';
```

how can do it ?

i try with splunk QL, logql, SQL but I can not do it.

or how search in json ?

help please

**Comment by wildwind123** [comment]
i do it with query "Splunk Ql"

```
index="*" | regex message="Watching movies.*" | fields message, host
```

but this query does not work, how ignore this symbol  ( " )  ?
```
index="*" | regex message="\"Watching movies.*" | fields message, host
```

how can select json value ?

**Reply by Macbeth98** [reply]
You can query it like this with Splunk QL.

```SPL
* | search message="*Watching movies*" | fields message, host
```

Or
```SPL
"*Watching movies*" | fields message, host
```

Or
```
* | regex message=".*Watching movies.*" | fields message, host
```

Let me know if the above queries are working.
